### PR TITLE
[Doc Improvement][App caching code snippet update][2933885]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-caching-for-your-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e50504f8791e8400ab11ada80e8915f17affa9f3/msteams-platform%2Fapps-in-teams-meetings%2Fapp-caching-for-your-tab-app.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fc406e8-09df-4b65-adf1-fd8327b06816) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-caching-for-your-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fe20cbc57cd19b02c9ea527e0533eb56cc932bef/msteams-platform%2Fapps-in-teams-meetings%2Fapp-caching-for-your-tab-app.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/12d4ef44-4531-4c0b-8158-4981e797769d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/app-caching-for-your-tab-app.md [hunk 1]
```diff
@@ -98,97 +98,24 @@ To enable app caching for your app, follow the steps:
...
+
+microsoftTeams.teamsCore.registerOnLoadHandler((data: any) => {
+    loadHandler(setItems, data);
+    setTitle("Entity Id : " + data.entityId);
+    console.log(data.contentUrl, data.entityId);
+    });
+
 ```
 
+To view the complete code sample, see [app caching in meeting](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/690da06cece8d2580bbab66ac466f4d4f192416e/samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx#L73).
```
Comment (Wajeed-msft) [inline msteams-platform/apps-in-teams-meetings/app-caching-for-your-tab-app.md:117]:
```suggestion
To view the complete code sample, see [app caching in meeting](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx#L73).
```

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-caching-for-your-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a0fbc7a7691bb6e4df476078dbe8db3160dd6d5/msteams-platform%2Fapps-in-teams-meetings%2Fapp-caching-for-your-tab-app.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/694a0ccc-8633-49a4-9d96-ca4c44f0fd67) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-caching-for-your-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e4f329f35340b205c8b2618b5484d1c305ae9070/msteams-platform%2Fapps-in-teams-meetings%2Fapp-caching-for-your-tab-app.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06131040-a465-4147-9b99-c56c50c6fb88) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**